quarkus-extension/simple-rest-example/src/main/java/org/acme/StartInterceptor.java

```diff
@@ -16,16 +16,12 @@
  */
 package org.acme;
 
-import org.camunda.bpm.engine.cdi.annotation.StartProcess;
 import org.camunda.bpm.engine.cdi.impl.annotation.StartProcessInterceptor;
 
 import javax.annotation.Priority;
 import javax.enterprise.context.Dependent;
-import javax.interceptor.Interceptor;
 
 @Priority(1)
 @Dependent
-@Interceptor
-@StartProcess
```

Comment:
This leads to registering the interceptor twice. We also need to change this in the test suite. With CAM-13790 I created a ticket for it.

Reply:
Actually, I was wrong here. I tested it with an old artifact I had in my local repository. I reverted the changes and deleted CAM-13790.